android/src/main/java/com/reactnativeweathersensormodule/WeatherSensorModulePackage.kt

```diff
@@ -0,0 +1,16 @@
+package com.reactnativeweathersensormodule
+import com.facebook.react.ReactPackage
+import com.facebook.react.bridge.NativeModule
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.uimanager.ViewManager
+
+
+class WeatherSensorModulePackage : ReactPackage {
+    override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
+        return listOf(WeatherSensorModule(reactContext))
+    }
+
+    override fun createViewManagers(reactContext: ReactApplicationContext): List<ViewManager<*, *>> {
+        return emptyList()
+    }
+}
```

Comment:
This file looks like it is using 4 space indentation, whereas the previous kt file is using 2. They should be consistent.